### ISSUE TITLE
Disk Stress 

### ISSUE THREAD

**Issue by thatsk** [ngine-io/chaotic#19]

**Comment by resmo** [comment]
we currently focus on experiments for cloud APIs.

Regarding nomad we could probably think about to implement a generic experiment executor for containers, in which you could run disk, network, etc, experiments.

I could think of prepared batch jobs which will get executed by chaotic.

What do you think about this idea?

**Comment by thatsk** [comment]
yeah that will be awesome the batch jobs with parameter will dispatch  job in terms of parameter we will pass

**Comment by thatsk** [comment]
here are some tools we can think about it.
https://github.com/alexei-led/pumba
https://github.com/ColinIanKing/stress-ng
